src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -208,14 +209,7 @@ def register_source_uri_arguments(scope):
 
 register_cli_argument('storage blob lease', 'blob_name', blob_name_type)
 
-for source in ['destination', 'source']:
-    register_cli_argument('storage blob copy', '{}_if_modified_since'.format(source), arg_group='Pre-condition')
-    register_cli_argument('storage blob copy', '{}_if_unmodified_since'.format(source), arg_group='Pre-condition')
-    register_cli_argument('storage blob copy', '{}_if_match'.format(source), arg_group='Pre-condition')
-    register_cli_argument('storage blob copy', '{}_if_none_match'.format(source), arg_group='Pre-condition')
-register_cli_argument('storage blob copy', 'container_name', container_name_type, options_list=('--destination-container', '-c'))
-register_cli_argument('storage blob copy', 'blob_name', blob_name_type, options_list=('--destination-blob', '-b'), help='Name of the destination blob. If the exists, it will be overwritten.')
-register_cli_argument('storage blob copy', 'source_lease_id', arg_group='Copy Source')
+register_cli_argument('storage blob copy start', 'blob_type', help=argparse.SUPPRESS, default='block')
```

Comment:
The actual copy operation only requires a source URI. This implies you could copy from a file share into a blob container. Then you would need to specify the blob type. We could infer the type based on the file extensions. VHD files default to page blobs and all others to block.  Something to consider.